cmd/collector/app/collector.go

```diff
@@ -16,9 +16,9 @@ import (
 	"github.com/jaegertracing/jaeger/cmd/collector/app/flags"
 	"github.com/jaegertracing/jaeger/cmd/collector/app/handler"
 	"github.com/jaegertracing/jaeger/cmd/collector/app/processor"
-	"github.com/jaegertracing/jaeger/cmd/collector/app/sampling/samplingstrategy"
 	"github.com/jaegertracing/jaeger/cmd/collector/app/server"
 	"github.com/jaegertracing/jaeger/internal/safeexpvar"
+	"github.com/jaegertracing/jaeger/internal/sampling/strategy"
```

Comment:
I suggest keeping the folder / package name as samplingstrategy (sorry if I said otherwise before), because it makes it more intuitive to use, i.e. `samplingstrategy.Provider` instead of `strategy.Provider`